editoast/editoast_derive/src/modelv2/codegen.rs

```diff
@@ -111,6 +113,24 @@ impl Identifier {
     }
 }
 
+impl ImplPlan {
```

Comment:
Because these functions are only useful in this module. I could have written `fn has_read(plan: &ImplPlan) -> bool` but I think it makes sense for these to be methods instead. Wdyt?

If they were defined in `args.rs`, they would need to be `pub(super)` at least, giving them more visibility than necessary.